app/assets/javascripts/app/mix_panel.js

```diff
@@ -45,6 +44,7 @@ App.addChild('MixPanel', {
     this.trackPageVisit('projects', 'new', 'Visited new project page');
     this.trackPageVisit('projects', 'index', 'Visited home');
     this.trackPageVisit('explore', 'index', 'Explored projects');
+    this.trackPageLoad('contributions', 'new', 'Started contribution');
```

Comment:
The problem here is that the logic to detect if we have a reward already selected is missing